integration-tests/features/steps/common.py

```diff
@@ -190,6 +191,11 @@ def finish_analysis_for_component(context, ecosystem, component, version):
         raise Exception('Timeout waiting for the component analysis results')
 
 
+def contains_alternate_node(json_resp):
+    """Check for the existence of alternate node in the stack analysis."""
+    return json_resp['result'][0].get('recommendations', None).get('alternate', None) is not None
```

Comment:
Thanks. I'll look at it. FYI - I just moved the code made by Sam to its own function w/o altering it. I'm updating it right now.